numpy/core/tests/test_half.py

```diff
@@ -69,6 +69,58 @@ def test_half_conversions(self):
         j = np.array(i_f16, dtype=int)
         assert_equal(i_int, j)
 
+    @pytest.mark.parametrize("offset", [None, "up", "down"])
+    @pytest.mark.parametrize("shift", [None, "up", "down"])
+    @pytest.mark.parametrize("float_t", [np.float32,])
```

Comment:
Also test with `np.float64`? Otherwise why parametrize it?

Reply:
Yeah, forgot to read after trying, hopefully the float64s fail ;) because of that 3 bit thing above.